src/Native/Unix/System.Security.Cryptography.Native/pal_rsa.c

```diff
@@ -93,7 +113,7 @@ CryptoNative_RsaPrivateDecrypt(int32_t flen, const uint8_t* from, uint8_t* to, R
 {
     if (HasNoPrivateKey(rsa))
     {
-        ERR_PUT_error(ERR_LIB_RSA, RSA_F_RSA_PRIVATE_DECRYPT, RSA_R_VALUE_MISSING, __FILE__, __LINE__);
+        ERR_put_error(ERR_LIB_RSA, RSA_F_RSA_NULL_PRIVATE_DECRYPT, RSA_R_VALUE_MISSING, __FILE__, __LINE__);
```

Comment:
The function code is not important anyway. `RSA_F_RSA_NULL_PRIVATE_DECRYPT` is present in both OpenSSL 1.0 and 1.1, BoringSSL dropped function codes altogether.